src/index.html

```diff
@@ -3,11 +3,38 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="card" data-qa="card">
+      <img
+      class="card__image"
+      src="./images/imac.jpeg"
+      alt="Apple iMac">
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__id-good">Product code: 195434</p>
+      <div class="rate">
```

Comment:
```suggestion
      <p class="card__id-good">Product code: 195434</p>
      
      <div class="rate">
```
checklist
<img width="948" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/e887d631-a761-4e90-a5c9-a587a94f5f91">

fix all such cases